openapi/api.yml

```diff
@@ -1013,6 +1013,10 @@ paths:
           description: Forbidden
         '404':
           description: Not Found
+        '422':
+          description: Unprocessable Entity
+        '500':
+          description: Generic Error
   '/v3/users/actions/validate-bo-reset-password/{id}':
     get:
       tags:
```

Comment:
considering also the other PRs pending and the number of endpoints present, wouldn't it be appropriate to purge the api `yaml` file to only actually used by `io-pay` ?

Reply:
@pasqualespica I agree but let's makit part of another story